db/migrate/20181122210927_add_letsencrypt_email_to_app.rb

```diff
@@ -0,0 +1,6 @@
+class AddLetsencryptEmailToApp < ActiveRecord::Migration[5.1]
+  def change
+    add_column :apps, :letsencrypt_email, :string
+    add_column :apps, :letsencrypt_enabled, :boolean
```

Comment:
Do we want to make this `false` by default?

Reply:
I thought about that, but that wouldn't express the case when servers are already LetsEncrypted and already present in an Intercity that would be updated with this feature. So I'm using `nil` for the "unknown" state for servers that are already in IC, but not having synced it's LE status.

`nil` = LE status unknown (will be updated on next fetch)
`true` = Was recognised enabled or enabled via our joob
`false` = Was recognised as disabled